modules/storages/app/common/storages/adapters/providers/sharepoint/queries/internal/lists_query.rb

```diff
@@ -75,7 +75,7 @@ def parse_response(json)
                     id: entry.dig(:drive, :id).to_s,
                     mime_type: "application/x-op-drive",
                     location: UrlBuilder.path("/", entry[:name]),
-                    permissions: %i[readable writeable]
+                    permissions: %i[readable]
```

Comment:
I do not understand why do we mark "files" in the root not-wrieable as well. Could you clarify this for me please?
In the bug description only root folder is mentioned.

Reply:
Yes. the root "folder" for sharepoint is the site (a list of drives). =)

Reply:
But why are you making drives read only as well?